packages/js/product-editor/src/products-app/product-form.stories.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { createElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import {
+	PRODUCT_FIELDS,
+	PRODUCT_FIELDS_KEYS,
+	ProductForm,
+	PRODUCTS_DATA,
+} from './utilites/storybook';
+
+export default {
+	title: 'Product App/Product Form',
+	component: ProductForm,
```

Comment:
Is there a reason you end up pushing both `ProductForm` and `ProductsView` to the utilities/storybook file? 
It feels slightly weird there, could we instead just add it to this `stories` file?

Reply:
I would expect that in the following weeks, we will replace `ProductForm` and `ProductsView` with the components used in the `products-app` project. These two are just components are just placeholder necessary to experiment with dataviews library without dealing with the entire products-app application.